metho-set.js

```diff
@@ -27,3 +27,54 @@ export const difference = Metho.add(
   }
 )
 
+// Map items
+export const map = Metho.add(
+  target,
+  function map(fn) {
+    return new Set([...this].map(i => fn(i)));
```

Comment:
Also pass the set itself into the callback function - to mirror the additional parameters that array method callbacks receive (obviously passing in `index` makes no sense for sets). This should apply to all of these new methods.

We can also drop the semi-colons... I'm not sure if I've been consistent with this in the Metho code, but I prefer to not have them

Reply:
Good point! Should I pass the value twice and then the set to be consistent with [Set#forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/forEach)?

Ah, the semicolons are an oversight. Will remove them!

Reply:
Yeah, that's a good idea for consistency 👍